networking/advertise-service-ips.md

```diff
@@ -49,10 +50,23 @@ If your {{site.prodname}} deployment is configured to peer with BGP routers outs
 
 ### Before you begin...
 
-- [Configure BGP peering]({{ site.baseurl }}/networking/bgp) between {{site.prodname}} and your network infrastructure
+**Required**
+- [Configure BGP peering]({{site.baseurl}}/networking/bgp) between {{site.prodname}} and your network infrastructure
 - For ECMP load balancing to services, the upstream routers must be configured to use BGP multipath.
 - You need at least one external node outside the cluster that acts as a router, route reflector, or ToR that is peered with calico nodes inside the cluster.
-- Services must be configured with the correct service type (“Cluster” or “Local”) for your implementation. For `externalTrafficPolicy: Local`, the service must be type `LoadBalancer` or `NodePort`.
+- Services must be configured with the correct service mode (“Cluster” or “Local”) for your implementation. For `externalTrafficPolicy: Local`, the service must be type `LoadBalancer` or `NodePort`.
+
+**Limitations**
+- OpenShift, versions 4.5 and 4.6  
+    There is a {% include open-new-window.html text='bug' url='https://github.com/kubernetes/kubernetes/issues/91374' %} where the source IP is not preserved by NodePort service with externalTrafficPolicy:Local or by ClusterIP service, and when using a LoadBalancer service. To work around this, you can upgrade to **OpenShift 4.7** (where bug is fixed), or use this {% include open-new-window.html text='workaround to avoid SNAT with ExternalIP' url='https://docs.openshift.com/container-platform/4.7/nodes/clusters/nodes-cluster-enabling-features.html' %}:
```

Comment:
```suggestion
    There is a {% include open-new-window.html text='bug' url='https://github.com/kubernetes/kubernetes/issues/91374' %} where the source IP is not preserved by NodePort services or traffic via a Service ExternalIP with externalTrafficPolicy:Local. 
    
OpenShift users on v4.5 or v4.6 can use this {% include open-new-window.html text='workaround to avoid SNAT with ExternalIP' url='https://docs.openshift.com/container-platform/4.7/nodes/clusters/nodes-cluster-enabling-features.html' %}:

  oc edit featuregates.config.openshift.io cluster
   spec:
     customNoUpgrade:
       enabled:
       - ExternalPolicyForExternalIP

Kubernetes users on version v1.18 or v1.19 can enable source IP preservation for NodePort services using the ExternalPolicyForExternalIP feature gate. 

Source IP preservation for NodePort and services and ExternalIPs is enabled by default in OpenShift v4.7+, and Kubernetes v1.20+. 
```

Reply:
Fixed.